src/calculateRentalCost.js

```diff
@@ -3,8 +3,30 @@
  *
  * @return {number}
  */
+const costRental = 40;
+const mediumTerm = 3;
+const mediumTermDiscound = 20;
+const longTerm = 7;
+const longTermDiscount = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days >= longTerm) {
+    result = days * costRental - longTermDiscount;
```

Comment:
The logic here applies the long-term discount if the number of days is greater than or equal to `longTerm`. This is correct, but ensure that the discount logic aligns with the task requirements.